lib/error-pages-router.js

```diff
@@ -0,0 +1,55 @@
+var route;
```

Comment:
shouldn't be a jag because it's not mapped form jaggery.conf . move to different jag or a js